test/bddtests/bddtests_test.go

```diff
@@ -46,20 +45,11 @@ func TestMain(m *testing.M) {
 	compose := os.Getenv("DISABLE_COMPOSITION") != "true"
 	status := godog.RunWithOptions("godogs", func(s *godog.Suite) {
 		s.BeforeSuite(func() {
-
 			if compose {
 				if err := context.Composition().Up(); err != nil {
 					panic(fmt.Sprintf("Error composing system in BDD context: %s", err))
 				}
-				fmt.Println("docker-compose up ... waiting for peer to start ...")
-				testSleep := 30
```

Comment:
I think we still need the sleep time

Reply:
No, I updated fabric-peer-test-common to do retries until it connects to the peer.